stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/info/SelectedChannelMenu.kt

```diff
@@ -139,23 +144,14 @@ internal fun DefaultSelectedChannelMenuHeaderContent(
 /**
  * Represents the default content shown at the center of [SelectedChannelMenu] dialog.
  *
- * @param selectedChannel The channel the user selected.
- * @param isMuted If the channel is muted for the current user.
  * @param onChannelOptionClick Handler for when the user selects a channel option.
+ * @param channelOptions The list of options to show in the UI, according to user permissions.
  */
 @Composable
 internal fun DefaultSelectedChannelMenuCenterContent(
-    selectedChannel: Channel,
```

Comment:
The function body is so simple now that it's not strictly needed, but I kept it as default functions are a part of the way we do things.

Reply:
Yeah it's fine, in case we want to extend this and customize the options UI a bit more or something, we have the ability to do so